agent.py

```diff
@@ -77,6 +77,7 @@ def _handle_sigusr1(self, signum, frame):
         self._handle_sigterm(signum, frame)
         self._do_restart()
 
+    @classmethod
     def info(self, verbose=None):
```

Comment:
Tiny tiny nitpick but if you're switching to a class method, it should be 
```
cls
```
instead of 
```
self
```